irpc.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python3
 
 import sys, copy
-from pycparser.c_ast import FuncCall, ID, Decl
+from pycparser.c_ast import *
+#FuncCall, ID, Decl, TypeDecl, IdentifierType, If
 
```

Comment:
Please import only what if needed and avoid *
By doing so, it's easier for a future programmer to understand where function comes from (It will not be obvious in the future that `If` come form pycparser.c_ast).